designsafe/static/scripts/projects/components/publication-download/publication-download.template.html

```diff
@@ -12,12 +12,23 @@ <h3 class="text-center">
     </h3>
 </div>
 <div class="modal-body" ng-if="$ctrl.loaded">
-    <div ng-if="!$ctrl.error" style="margin-top: 5px;">
+    <div ng-if="!$ctrl.error && $ctrl.arcData.length <= 2147483648" style="margin-top: 5px;">
         <p>
             This download is a ZIP file of the complete project dataset. The size of the ZIP file is <strong>{{$ctrl.arcData.length|bytes}}</strong>.
             <!-- A JSON file is included that contains all of the metadata. -->
         </p>
     </div>
+    <div ng-if="!$ctrl.error && $ctrl.arcData.length > 2147483648" style="margin-top: 5px;">
+        <p>
+            This project zipped is <strong>{{$ctrl.arcData.length|bytes}}</strong>, exceeding the <strong>2 GB</strong> download limit. To download, 
+            <a href="https://accounts.tacc.utexas.edu/register" target="_blank" aria-describedby="msg-open-new-window">create an account</a>
+            and follow the 
+            <a href="/rw/user-guides/data-transfer-guide/" target="_blank" aria-describedby="msg-open-new-window">Data Transfer Guide</a>. 
```

Comment:
There's an updated url for the data transfer guide that should be used here:
/user-guide/managingdata/#data-transfer-guides
the /rw paths are older paths when the guides were in cms, which they still technically are there, they're just old and out of date.

Reply:
Thanks. I updated the URL